### PR TITLE
`General`: Add documentation for iOS client

### PR DESCRIPTION
**Pull request by sven0311**
### Motivation and Context

This PR adds documentation on how to use the [Artemis iOS client](https://github.com/ls1intum/artemis-ios)

### Description

This PR only adds documentation and does not touch code at all.

### Steps for Testing

1. Take a look at the code documentation.
2. Inspect the User Guide -> Mobile Applications -> iOS application page and read the documentation carefully

#### Code Review
- [x] Docu Review 1
- [x] Docu Review 2

### CODE REVIEW

**Review by TimOrtel — APPROVED** [review]
Good extensions of the existing documentation.

**Review by terlan98 — DISMISSED** [review]
Documentation changes look good 👍

**Review by nityanandaz — APPROVED** [review]

**Review by nityanandaz — APPROVED** [review]
Great

File: docs/user/mobile-applications.rst [hunk 1]
```diff
@@ -93,13 +93,119 @@ Problems
...
+
+The iOS application supports the following features:
+
+#. Login:
+    #. Select predefined and custom Artemis servers
+    #. Login with username + password
+
+.. raw:: html
+
+    <iframe src="https://live.rbg.tum.de/w/artemisintro/35200?video_only=1&t=0" allowfullscreen="1" frameborder="0" width="600" height="350">
```
Comment (nityanandaz) [inline docs/user/mobile-applications.rst:107]: The iframe does not load for me in Safari and Firefox, can you render a fallback link?

**Review by terlan98 — APPROVED** [review]
The recent changes look good. The videos make the documentation easier to understand 👍

**Review by nityanandaz — APPROVED** [review]

**Review by Mtze — APPROVED** [review]

**Pull request merged**